libraries/Microsoft.Bot.Builder.Dialogs/Memory/Scopes/DialogMemoryScope.cs

```diff
@@ -82,7 +82,7 @@ public override void SetMemory(DialogContext dc, object memory)
                 dc.ActiveDialog.State = (IDictionary<string, object>)memory;
             }
 
-            throw new Exception("Cannot set DialogMemoryScope. There is no active dialog dialog or parent dialog in the context");
+            throw new InvalidOperationException("Cannot set DialogMemoryScope. There is no active dialog dialog or parent dialog in the context");
```

Comment:
We could fix this typo
```suggestion
            throw new InvalidOperationException("Cannot set DialogMemoryScope. There is no active dialog or parent dialog in the context");
```

Reply:
Looks good, I added the suggestion. Thanks!